src/adaptix/_internal/morphing/facade/provider.py

```diff
@@ -439,3 +441,35 @@ def default_dict(pred: Pred, default_factory: Callable) -> Provider:
     :param default_factory: default_factory parameter of the ``defaultdict`` instance to be created by the loader
     """
     return bound(pred, DefaultDictProvider(default_factory))
+
+
+def datetime_timestamp_provider(pred: Pred, tz: Optional[timezone] = None) -> Provider:
+    """Provider that can load/dump datetime object from/to UNIX timestamp.
+
+    :param pred: Predicate specifying where the provider should be used.
+        See :ref:`predicate-system` for details.
+    :param tz: tz parameter which will be passed to the datetime.fromtimestamp method.
+    """
+
+    return bound(pred, DatetimeTimestampProvider(tz))
+
+
+def datetime_format_provider(pred: Pred, fmt: str) -> Provider:
+    """Provider that can load/dump datetime object from/to format string e.g "%d/%m/%y %H:%M"
+
+    :param pred: Predicate specifying where the provider should be used.
+        See :ref:`predicate-system` for details.
+    :param fmt: format parameter which will be passed to datetime.strptime method.
+    """
+    return bound(pred, DatetimeFormatProvider(fmt))
+
+
+def date_timestamp_provider(pred: Pred) -> Provider:
+    """Provider that can load date object from UNIX timestamp.
+    Note that date objects can`t be dumped to the UNIX timestamp
+
+    :param pred: Predicate specifying where the provider should be used.
+        See :ref:`predicate-system` for details.
+    """
+    return bound(pred, DateTimestampProvider())
```

Comment:
This names are inconsistent with other names of public factories.
I think that ``datetime_by_timestamp``, ``datetime_by_format``, ``date_by_timestamp`` will look better.